eris.go

```diff
@@ -72,10 +74,35 @@ func wrap(err error, msg string) error {
 			root.stack.insertPC(*stack)
 		}
 	default:
+		// attempt to unwrap external errors while building a new error chain or fallback to flattening them
+		var errStr []string
+		for e != nil {
+			str := e.Error()
+			errStr = append([]string{str}, errStr...)
+			e = Unwrap(e)
```

Comment:
Instead of assuming other libs, is it possible to check for pkg errors by using reflect type?

Reply:
why wouldn't we want it to work for other packages too?